xroad-catalog-lister/README.md

```diff
@@ -10,6 +10,49 @@ A class diagram illustrating X-Road Catalog Lister implementation with the `defa
 See also the [Installation Guide](../doc/xroad_catalog_installation_guide.md) and
 [User Guide](../doc/xroad_catalog_user_guide.md).
 
+## Configuration
+
+X-Road Catalog Lister can be configured by creating a copy of the [application.yaml](/src/main/resources/application.yaml) file.
+The following properties are available:
+
+### Spring Boot Configurations
```

Comment:
Consider removing properties that the user is not expected to change. For example, the application supports only PostgreSQL so there's no need tell how to change dialect to support some other DB.